templates/cisco_ios_traceroute.textfsm

```diff
@@ -0,0 +1,44 @@
+Value Required,Filldown HOP_NUM (\d+)
+Value ADDRESS ((?:\d{1,3}\.){3}\d{1,3})
+Value FQDN (\S+)
+# Captures RTT (in milliseconds), or faults, like
```

Comment:
Would it be worth adding a DESTINATION value to this? So then it is clear as to where the traceroute is attempting to go?

Reply:
Why? This comes from the command, i.e. if someone traceroutes to 8.8.8.8, they know the destination well in advance. It's pointless value IMHO as it is known in advance.

Reply:
My take is that the destination is also stored in the data structure and can be used in further processing.

It's not a big ask, but I'm not 100% tied to it so if you want to leave it out, then that is fine.